website/docs/index.html.markdown

```diff
@@ -25,17 +25,66 @@ Use the navigation to the left to read about the available resources.
 | `region`               | Required | The region for the data center for which your New Relic account is configured. The `NEW_RELIC_REGION` environment variable can also be used. Valid values are `US` or `EU`. |
 | `insecure_skip_verify` | Optional | Trust self-signed SSL certificates. If omitted, the `NEW_RELIC_API_SKIP_VERIFY` environment variable is used.                                                               |
-| `insights_insert_url`  | Optional | Your Insights insert key used when inserting Insights events via the `newrelic_insights_event` resource. Can also use `NEW_RELIC_INSIGHTS_INSERT_KEY` environment variable. |
+| `insights_insert_key`  | Optional | Your Insights insert key used when inserting Insights events via the `newrelic_insights_event` resource. Can also use `NEW_RELIC_INSIGHTS_INSERT_KEY` environment variable. |
 | `cacert_file`          | Optional | A path to a PEM-encoded certificate authority used to verify the remote agent's certificate. The `NEW_RELIC_API_CACERT` environment variable can also be used.              |
 
 
+## Authentication Requirements
+
+This provider is in the midst of migrating away from our older REST based APIs
+to a newer GraphQL based API that we lovingly call NerdGraph.  During this
+transition, the provider will be using different endpoints depending on which
+resource is in use.  Below is a table that reflects the current state of the
+resources compared to which endpoint is in use.
+
+### Resources
+
+| Resource                                            | Endpoint  | Authentication        | Roles        |
+| --------------------------------------------------- | --------- | --------------------- | ------------ |
+| `newrelic_alert_channel`                            | RESTv2    | `api_key`             | Admin, Owner |
+| `newrelic_alert_condition`                          | RESTv2    | `api_key`             | Admin, Owner |
+| `newrelic_alert_policy`                             | NerdGraph | `api_key`             | Admin, Owner, Alerts manager |
+| `newrelic_alert_policy_channel`                     | RESTv2    | `api_key`             | Admin, Owner |
+| `newrelic_application_settings`                     | RESTv2    | `api_key`             | Admin, Owner |
+| `newrelic_dashboard`                                | RESTv2    | `admin_api_key`       | Admin, Owner |
+| `newrelic_entity_tags`                              | NerdGraph | `api_key`             | Admin, Owner |
+| `newrelic_events_to_metrics_rule`                   | NerdGraph | `api_key`             | Admin, Owner |
+| `newrelic_infra_alert_condition`                    | RESTv2    | `api_key`             | Admin, Owner |
+| `newrelic_insights_event`                           | RESTv2    | `insights_insert_key` | Admin, Owner, Insights manager |
+| `newrelic_nrql_alert_condition`                     | NerdGraph | `api_key`             | Admin, Owner, Alerts manager |
```

Comment:
@jthurman42 Is roles here the required role for managing this resource?

Reply:
Roles that can manage the resources, I initially had `Required Role` but changed it to the shorter version.  Any of those roles / add-on roles would be able to manage the resources.